### PR TITLE
feat(utils): make release script

### LINKED ISSUE #3294: Create `yarn release` script

### PR DESCRIPTION
**Pull request by asiaznik**
Closes #3294 

# Description ℹ️

Trigger a release by running `yarn release`

# Demo 📺

https://user-images.githubusercontent.com/1980305/228259772-f837a02a-e000-4534-a0a7-d45837d4dbfc.mov

### CODE REVIEW

**Review by mattrussell36 — CHANGES_REQUESTED** [review]
Can you move make-release.js to the scripts directory plz

**Comment by mattrussell36** [comment]
@asiaznik It would also be good to update the notion page on releases to contain information about the script 🙏

**Comment by asiaznik** [comment]
> @asiaznik It would also be good to update the notion page on releases to contain information about the script 🙏

https://www.notion.so/vegaprotocol/Tagging-releases-and-deployment-f61517d6e7c6488ea0606e48f11b489a?pvs=4#701646c1240441a18b4406ee932db864

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "build:all": "nx run-many --all --target=build",
     "lint:all": "nx run-many --all --target=lint",
     "e2e:all": "nx run-many --all --target=e2e",
-    "vegacapsule": "vegacapsule network bootstrap --config-path=../frontend-monorepo/vegacapsule/config.hcl"
+    "vegacapsule": "vegacapsule network bootstrap --config-path=../frontend-monorepo/vegacapsule/config.hcl",
+    "release": "git checkout develop ; git pull ; node scripts/make-release.js"
```
Comment (macqbat) [inline package.json:15]: mainnet shoudn't be released from master?

File: scripts/make-release.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        } to ${answers.envs.join(', ')}`,
+      })
+      .then(() => {
+        for (const env of answers.envs) {
+          release(answers.tag, env);
+        }
+        execSync('git checkout develop');
+      })
+      .catch((err) => {
+        console.log('Something went wrong');
```
Comment (macqbat) [inline scripts/make-release.js:76]: log err.message or err.toString() would be useful

File: scripts/make-release.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      });
+  })
+  .catch((err) => {
+    console.log('Something went wrong');
```
Comment (macqbat) [inline scripts/make-release.js:80]: here as well

File: package.json (hunk 1 above)
Comment (asiaznik) [reply package.json:15]: true, but the tags are on develop anyway - added "mainnet" option to the script.

**Review by mattrussell36 — APPROVED** [review]
Happy to merge as is, but it might be good to make a couple small changes so that rather than immediately resetting the branch we open a PR instead. We might need a github action for it

**Pull request merged**